linux_os/guide/system/software/updating/ensure_GPG_keys_are_configured/rule.yml

```diff
@@ -0,0 +1,124 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure GPG keys are configured'
+
+description: |-
+    Most distributions use a package manager such as yum, apt, or zypper to install and 
+    update software packages. They use their own GPG keys directly or at least ship them
+    installed with the minimal system.  But third party packages signed by a different 
+    GPG key need manual handling. The operation system or installed application can be 
+    successfully bootstrapped without the GPG key being trusted. However, you cannot 
+    install new packages or update them until the keys are trusted.
+    <br /><br /> 
+    Trusting a GPG key is important for security on operating systems or applications. 
+    It is the task of the administrator to decide which keys are needed and can be trusted.
+    <br /><br />
+    Most packages managers implement GPG key signing to verify package integrity during
+    installation.
+    <br /><br />
+    To verify GPG keys are configured correctly for your package manager, 
+    one of the following command groups may provide the needed information
+    depending on the package manager in use. 
+    <br /><br />
+    In RedHat,Fedora,CentOs, and SUSE Linux distributions, the administrators have to 
+    follow the next steps: <br />
+    1. Log on to the system as a user with administrator rights.<br />
+    2. Locate and download package, for example zoom_x86_64.rpm<br />
+    3. Locate and download the public key (GPG) from the software download site, for example 
+    the key for zoom package is package-signing-key-5-12-6.pub<br />
+    4. Import the key public key: <br />
+    <tt>$ sudo rpm --import package-signing-key-5-12-6.pub</tt><br />
+    5. List the keys, for example the comman d<br /> 
+    <tt>$ sudo rpm -qa gpg-pubkey*</tt><br />
+    will provide:<br />
+    <tt>gpg-pubkey-dd79b481-62fe7502</tt><br />
+    6. Get more details about the key, via the command: <br />
+    <tt>$ sudo rpm -qa gpg-pubkey-dd79b481-62fe7502</tt><br />
+    7. Check the GPG key, for example the command: <br />
+    <tt>$ sudo rpm --checksig zoom_x86_64.rpm</tt><br />
+    will provide: <br />
+    <tt>zoom_x86_64.rpm: digests signatures OK</tt><br />
+    8. Check the GPG key in another way, for example the command: <br /> 
```

Comment:
I believe one approach should be recommended instead of two. Which one is more precise? If both are precise, which one provides a better user experience?